### PR TITLE
[LL API] Add a test for scoring

### PR DESCRIPTION
**Pull request by ashtul**
Related to RedisGraph/RedisGraph#1244

### CODE REVIEW

File: src/redisearch_api.c [hunk 1]
```diff
@@ -439,7 +439,7 @@ int RediSearch_DocumentExists(IndexSpec* sp, const void* docKey, size_t len) {
 }
 
 RS_ApiIter* RediSearch_IterateQuery(IndexSpec* sp, const char* s, size_t n, char** error) {
-  QueryInput input = {.qtype = QUERY_INPUT_STRING, .u = {.s = {s, .n = n}}};
```
Comment (oshadmi) [inline src/redisearch_api.c:442]: Strange that it does not work as expected (`qs` is first and initialised by order and `n` is initialised using `.n` designator)
We should probably search if we have similar occurrences of the same kind?

**Review by oshadmi — DISMISSED** [review]

File: src/redisearch_api.c (hunk 1 above)
Comment (oshadmi) [reply src/redisearch_api.c:442]: Maybe using
egrep '{[a-z0-9()_]*[.][a-z0-9_]+ *= *[a-z0-9()_]+'
to find lines with initialization using dot designators, and then searching in the result for initializations without designators that precede them?

Comment (ashtul) [reply src/redisearch_api.c:442]: @oshadmi can we approve this PR? This comment is not really related to it.

**Review by oshadmi — APPROVED** [review]
👍🏼

**Pull request merged**